git-config/hooks/pre-commit

```diff
@@ -2,8 +2,8 @@
 set -o errexit -o nounset
 
 
-forbidden_hosts=("ptah" "sobek" "nu" "ub16" "ub15" "ub27" "ub28")
-short_host=$(hostname --short)
+readonly forbidden_hosts=("ptah" "sobek" "nu" "ub16" "ub15" "ub27" "ub28")
```

Comment:
by the way, before we forget it: can you please remove ub27 (no ub_tools installation there) and add ub53 ub54 ub55? These will be our Ubuntu Server replacements for ub28, ub15+ub16 (please leave the old ones in there for now.)

Reply:
or maybe we just remove this section entirely... since mid 2021, you can only push with a ssh key which will not be installed on the machines anyway.

Reply:
So, should I remove it?